### ISSUE TITLE
Value 'Extreme' at 'quality' failed to satisfy constraint

### ISSUE THREAD

**Issue by Nazrax** [mkb79/audible-cli#107]
I've used audible-cli for ages, but when I tried to download a book today I got a weird new error:

```$ audible-cli download --aaxc -t "Emperor's Soul"

Found the following matches for 'Emperor's Soul'
+---------+--------------------+------------+
| % match | title              |    asin    |
+---------+--------------------+------------+
|  100.0  | The Emperor's Soul | B009XEKR3O |
+---------+--------------------+------------+
Proceed with this audiobook(s) [y/N]: y
Error in job: Bad Request (400): 1 validation error detected: Value 'Extreme' at 'quality' failed to satisfy constraint: Member must satisfy enum value set: [High, Normal]
```

I tried other books, and they all gave the same results. `--aax` works, but some books can only be downloaded as aaxc. `-q high` gives the same error, but `-q normal` seems to work.

After receiving the error I updated to the latest version of audible-cli, but the error is still present.

**Comment by mkb79** [comment]
This issue is fixed in the master branch yesterday. But I didn’t release a new version yet. There are some other changes in the master branch since the last version release. When I have make sure they are running without any new issues, I will release this. 

If you can’t wait, please install `audible-cli` from the GitHub master branch. So you can help testing and speed up the new release 😊!

**Comment by Nazrax** [comment]
I thought I'd pulled and installed the latest master, but I must have goofed something, because after a fresh installation I can see the updated version.

That's quite a fancy interface; though if you're looking for feedback, you may want to put the `(Use arrow keys ...` on a second line, as it has a rather ugly wrap on my 120 column terminal.

As to the books, they downloaded with no issues. Thanks!

**Comment by CallumKerrEdwards** [comment]
I am also seeing this, and updating to the latest on master fixed the download 👍

**Comment by mkb79** [comment]
Released v0.2.1 which solves this issue
